### DISCUSSION TITLE
Existe função para realizar transferência de valores via PIX?

### DISCUSSION THREAD

**Discussion by tarciziovn** [andreroggeri/pynubank#359] [Q&A] [answered]
Ao consultar a lista de funções presentes em nubank.py não encontrei nenhuma função cujo nome intuitivamente me parecesse ser algo que realizasse uma transação de transferência de valores via PIX utilizando a API.
Desta forma, gostaria de saber se esta função está implementada em algum outro local ou se existe perspectiva no roadmap de desenvolvimento para implementar esta funcionalidade.

**Comment by andreroggeri** [comment, marked as answer]
Essa funcionalidade não existe e não vejo como fazer isso.

Atualmente fazemos o processo de login do app, e sempre que você conecta um aparelho novo é necessário fazer a segunda validação (Que você tira uma selfie e envia para eles).

Sem fazer a segunda validação, o token e certificado são somente leitura, não sendo possível fazer operações na conta (TED, Pix, Pagar boleto, etc).

Eu não tenho intenção de implementar isso por não precisar e também achar perigoso (Visto que já encontrei alguns repos com CPF/Senha commitados)
